src/cli/utils/sharepoint.go

```diff
@@ -0,0 +1,97 @@
+package utils
+
+import (
+	"errors"
+
+	"github.com/alcionai/corso/src/pkg/selectors"
+)
+
+const (
+	LibraryItemFN = "library-item"
```

Comment:
@gmatev not sure what term we want to use for items here.  `library-item` is accurate, but a mouthful.  Granted, in the case of sharepoint, I expect users will much more rarely interact with restores on a per-item basis, so perhaps that's okay?